src/Carousel/Carousel.d.ts

```diff
@@ -0,0 +1,49 @@
+
+// WARNING do not make manual Changes to this file.
+// widget.d.ts files is auto generated from the params in the widget.xml
```

Comment:
Do not commit generated files. Generate them whenever they are needed. This is a definition file which is only used by the typescript compiler. It's not needed at runtime.